src/pyOpenMS/pxds/Acquisition.pxd

```diff
@@ -1,18 +1,26 @@
 from libcpp cimport bool
 from MetaInfoInterface cimport *
+from MetaInfoRegistry cimport *
+from Types cimport *
+from DataValue cimport *
+from String cimport *
 
 cdef extern from "<OpenMS/METADATA/Acquisition.h>" namespace "OpenMS":
-    
-    cdef cppclass Acquisition(MetaInfoInterface):
-        # wrap-inherits:
-        #    MetaInfoInterface
+	
+	cdef cppclass Acquisition(MetaInfoInterface):
+		# wrap-inherits:
+		#    MetaInfoInterface
 
-        Acquisition() nogil except +
-        Acquisition(Acquisition) nogil except +
-
-        bool operator==(Acquisition &rhs) nogil except +
-        bool operator!=(Acquisition &rhs) nogil except +
-        String getIdentifier() nogil except +
-        void setIdentifier(const String &identifier) nogil except +
+		Acquisition() nogil except +
+		Acquisition(Acquisition) nogil except +
 
+		bool operator==(Acquisition &rhs) nogil except +
+		bool operator!=(Acquisition &rhs) nogil except +
+		String getIdentifier() nogil except +
+		void setIdentifier(const String &identifier) nogil except +
 
+		void getKeys(libcpp_vector[String] & keys)
+		DataValue getMetaValue(String) nogil except +
+		void setMetaValue(String, DataValue) nogil except +
+		bool metaValueExists(String) nogil except +
+		void removeMetaValue(String) nogil except +
```

Comment:
do you need those? they should already be here through inheritance (see https://github.com/OpenMS/OpenMS/blob/develop/src/pyOpenMS/pxds/MetaInfoInterface.pxd) --

Reply:
Yes, they are needed. The inheritance does not seem to transpire through cython. I also built the recent master as pyopenms, same there. 
```
>>> exp[0].getAcquisitionInfo().size()
1
>>> exp[0].getAcquisitionInfo()[0].isMetaEmpty()
True
>>> exp[0].getAcquisitionInfo()[0].setMetaValue(b"key", 42)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'pyopenms.pyopenms_6.Acquisition' object has no attribute 'setMetaValue'
>>> exp[0].getAcquisitionInfo()[0].getMetaValue(b"key")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'pyopenms.pyopenms_6.Acquisition' object has no attribute 'getMetaValue'
```
I have no explanation, but if you need more convincing, look at the scope of the 'fixed' Acquisition vs the scope of 'unfixed' AcquisitionInfo (that also has MetaInfoInterface inheritance):
```
>>> dir( exp[0].getAcquisitionInfo()[0] )
['__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '_init_0', '_init_1', 'clearMetaInfo', 'getIdentifier', 'getKeys', 'getMetaValue', 'isMetaEmpty', 'metaRegistry', 'metaValueExists', 'removeMetaValue', 'setIdentifier', 'setMetaValue']
>>> dir( exp[0].getAcquisitionInfo() )
['__class__', '__delattr__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__init__', '__le__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '_init_0', '_init_1', 'clearMetaInfo', 'getMethodOfCombination', 'isMetaEmpty', 'metaRegistry', 'push_back', 'resize', 'setMethodOfCombination', 'size']
```
:arrow_right: get/set is missing, maybe that gives some clue what is going on.

Reply:
strange, this seems to work here
```
>>> import pyopenms
>>> pyopenms.Acquisition().setMetaValue("test", 2)
```

Reply:
What's even stranger is that 'isMetaEmpty' and 'metaRegistry' is in scope. I took a deeper look (took some time to make the containers), and this is the case for the Release2.4.0 tag, but it is fine for develop.
Since the inheritance did not change between then and now, I am at a loss. :man_shrugging:  
Looking forward, I don't think I should leave it in when it does work w/o in the current develop.

Reply:
gone.